src/main/java/net/mcreator/ui/help/HelpLoader.java

```diff
@@ -95,20 +101,29 @@ public static String loadHelpFor(IHelpContext helpContext) {
 
 			if (helpContext.getEntry() != null) {
 				String helpText = getFromCache(helpContext.getEntry());
-				if (helpText != null) {
+				if (helpText != null && (helpText.contains("${") || helpText.contains("<#"))) {
 					if (helpContext instanceof ModElementHelpContext meHelpContext) {
 						try {
-							GeneratableElement generatableElement = meHelpContext.getModElementFromGUI();

```

Comment:
@KlemenDEV I put this additional assignment to prevent any potential differences between GEs taken from GUI at different moments of time (e. g. if you change spinner's value that triggers other elements state update I'd say).
If you had a reason for this, everything is fine ;)

Reply:
Help context is loaded in UI thread so all should be good. Good thinking nevertheless, glad you think out of the box and forsee such scenarios :)

Reply:
Thanks :)
So do I change it back?

Reply:
No need, it is safe in this case to access multiple times